package/camera-control/camera-control.mk

```diff
@@ -0,0 +1,24 @@
+################################################################################
+#
+# camera-control
+#
+################################################################################
+
+CAMERA_CONTROL_VERSION = d7f5e46f5f0f9f53d3fece9f563ad5a019e9ca70
+CAMERA_CONTROL_SITE = git://github.com/peterbay/camera-control.git
+CAMERA_CONTROL_LICENSE = GPL-3.0+
+CAMERA_CONTROL_LICENSE_FILES = LICENSE
+CAMERA_CONTROL_DEST_DIR = /opt/camera-control
+CAMERA_CONTROL_SITE_METHOD = git
```

Comment:
You need `CAMERA_CONTROL_DEPENDENCIES` to be `ncursesw` or something of that tune here, or this package will fail to build from time to time due to this package being built before `ncurses`.

Reply:
Thought the external packages will always be last, but good catch!